### PR TITLE
Offer to fix all properties to use a block body if they contain a throw expressions prior to C#7

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/20362

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

File: src/EditorFeatures/CSharpTest/UseExpressionBody/Analyzer/UseExpressionBodyForPropertiesAnalyzerTests.cs [hunk 1]
```diff
@@ -365,5 +365,64 @@ int Foo(int i)
...
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExpressionBody)]
+        public async Task TestOfferToConvertToBlockEvenIfExpressionBodyPreferredIfHasThrowExpressionPriorToCSharp7_FixAll()
+        {
+            await TestAsync(
+@"
+using System;
+class C
+{
+    int Foo {|FixAllInDocument:=>|} throw new NotImplementedException();
+    int Bar => throw new NotImplementedException();
```
Comment (sharwell) [inline src/EditorFeatures/CSharpTest/UseExpressionBody/Analyzer/UseExpressionBodyForPropertiesAnalyzerTests.cs]: :bulb: Add a third property to verify that the fix all only targets the error conditions:

```csharp
int StaysAsIs => 0;
```

Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/UseExpressionBody/Analyzer/UseExpressionBodyForPropertiesAnalyzerTests.cs]: will do.

**Comment by CyrusNajmabadi** [comment]
Fix added here: https://github.com/dotnet/roslyn/pull/20844

**Pull request merged**